.github/workflows/vast.yaml

```diff
@@ -1096,3 +1096,43 @@ jobs:
       - name: Run unit tests
         working-directory: python
         run: poetry run pytest
+      - name: Build package
+        working-directory: python
+        run: poetry build
+
+  python-package:
+    needs:
+      - python
+      - determine-version
+    if: github.event_name == 'release'
+    name: Python Package
+    runs-on: ubuntu-latest
+    env:
+      DEBIAN_FRONTEND: noninteractive
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref }}
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Setup Poetry image
+        uses: abatilo/actions-poetry@v2.1.6
+        with:
+          poetry-version: 1.2.1
+      - name: Configure Test PyPI
+        working-directory: python
+        run: |
+          poetry config repositories.test-pypi https://test.pypi.org/legacy/
+          poetry config pypi-token.test-pypi "${{ secrets.TEST_PYPI_TOKEN }}"
```

Comment:
I noticed that this is a new secret. For new secrets, please manage them as [Organization Secrets](https://github.com/organizations/tenzir/settings/secrets/actions), and enable them for the tenzir/vast repository only. This has the advantage that we can see all secrets in one place, and enable them for individual repositories selectively.